### PR TITLE
Custom Footer HTML Translations

### PR DESCRIPTION
**Pull request by afrastgeek**
Related to flarum/core#1267 and flarum/core#1315

Before, it was in flarum/flarum-ext-english#111 .

I thought it has been merged, so i update my fork with this repo head, which automatically close the MR. And then i see it wasnt merged yet. So, i recreate the MR here.

### CODE REVIEW

File: locale/core.yml [hunk 1]
```diff
@@ -19,12 +19,15 @@ core:
       colored_header_label: Colored Header
       colors_heading: Colors
       colors_text: "Choose two colors to theme your forum with. The first will be used as a highlight color, while the second will be used to style background elements."
+      custom_footer_heading: Custom Footer
+      custom_footer_text: "Add HTML to be displayed at the very bottom of the page."
```
Comment (franzliedke) [inline locale/core.yml]: Please order the keys alphabetically. ;)

Comment (afrastgeek) [reply locale/core.yml]: sure, thing. will change it asap

Comment (franzliedke) [reply locale/core.yml]: Almost. You forgot these ones... ;)

Comment (afrastgeek) [reply locale/core.yml]: I'm sorry. But can you help me figure what's i miss?

I've tried to sort it using Sublime's sort function. But nothing happen :(

Comment (franzliedke) [reply locale/core.yml]: Uhm, must have been a bug in my brain. :confused:

Comment (afrastgeek) [reply locale/core.yml]: no worries 😄

[the remark this answers is not in the document]
Comment (afrastgeek) [reply locale/core.yml]: good, catch @milescellar 
sorry that i miss it.

anyway, thanks for the fix on the other MR.

**Pull request merged**